handler/src/main/java/io/netty/handler/ssl/JdkAlpnApplicationProtocolNegotiator.java

```diff
@@ -20,21 +20,11 @@
 /**
  * The {@link JdkApplicationProtocolNegotiator} to use if you need ALPN and are using {@link SslProvider#JDK}.
  */
+// TODO(nmittler): Can we make this package-private?
```

Comment:
I think we should just leave it as it is for now... There is not much gain in doing this change.